src/services/healthcareProfessionalService.ts

```diff
@@ -92,74 +94,98 @@ function mapDbEntityTogqlEntity(dbEntity : firebase.DocumentData) {
 }
 
 function mapAndValidateDegrees(degreesInput: typeDefs.Degree[]) {
-    const degrees = degreesInput.map((degree: typeDefs.Degree) => {
-        const newDegree = {nameJa: degree.nameJa,
-            nameEn: degree.nameEn,
-            abbreviation: degree.abbreviation}
-
-        return newDegree
-    })
-
-    return degrees
+    try {
```

Comment:
Improve error handling in API

For handling errors, we should put try/catches in each of the resolvers.  Then if any methods expect an error, write nice handling logic.